python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,131 @@
+package {{ package_name }};
+
+import java.util.Objects;
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
+import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.exceptions.CfnResourceConflictException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class {{ operation }}Handler extends BaseHandlerStd {
+    private static final String CREATED_RESOURCE_STATUS = "available";
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<ServiceSdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
```

Comment:
I think we should add more comments on explaining what is ProgressEvent

Reply:
since `ProgressEvent` is framework agnostic (used in both OOP and FP) then would it make sense to put in public docs and not in the codegen stubs?

Reply:
We could put JavaDocs on the `ProgressEvent` class if it's not there, but otherwise don't put the comments in here I don't think.